### PR TITLE
chore(storage-browser): remove composability of Refresh

### PR DESCRIPTION
**Pull request by hbuchel**
#### Description of changes

Remove composability of Refresh

#### Checklist

- [ ] Have read the [Pull Request Guidelines](https://github.com/aws-amplify/amplify-ui/blob/main/CONTRIBUTING.md)
- [ ] PR description included
- [ ] `yarn test` passes and tests are updated/added
- [ ] PR title and commit messages follow [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/#summary) syntax
- [ ] _If this change should result in a version bump_, [changeset added](https://github.com/changesets/changesets/blob/main/docs/adding-a-changeset.md) (This can be done after creating the PR.) This does not apply to changes made to `docs`, `e2e`, `examples`, or other private packages.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by timngyn — APPROVED** [review]

File: packages/react-storage/src/components/StorageBrowser/createStorageBrowser.tsx [hunk 1]
```diff
@@ -54,14 +54,3 @@ export function createStorageBrowser<
...
-// below is functioning as a "type canary", will start yelling if `elements`
-// inference is broken
-const {
-  StorageBrowser: {
-    LocationsView: { Controls },
-  },
-} = createStorageBrowser({ elements: { Button } });
-const _SillyTest = () => (
-  <Controls.Refresh.Button alignContent={'-moz-initial'} />
-);
```
Comment (calebpollman) [inline packages/react-storage/src/components/StorageBrowser/createStorageBrowser.tsx:67]:
```suggestion
```

File: packages/react-storage/src/components/StorageBrowser/createStorageBrowser.tsx [hunk 2]
```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import { Button } from '@aws-amplify/ui-react';
+// import { Button } from '@aws-amplify/ui-react';
```
Comment (calebpollman) [inline packages/react-storage/src/components/StorageBrowser/createStorageBrowser.tsx:2]:
```suggestion
```

**Review by calebpollman — APPROVED** [review]
LGTM. Added some suggestions to prevent merge conflicts with https://github.com/aws-amplify/amplify-ui/pull/5484

File: packages/react-storage/src/components/StorageBrowser/createStorageBrowser.tsx (hunk 2 above)
Comment (hbuchel) [reply packages/react-storage/src/components/StorageBrowser/createStorageBrowser.tsx]: Thanks updated these in latest

File: packages/react-storage/src/components/StorageBrowser/createStorageBrowser.tsx (hunk 1 above)
Comment (hbuchel) [reply packages/react-storage/src/components/StorageBrowser/createStorageBrowser.tsx]: Updated in latest to fix merge conflicts

**Review by calebpollman — APPROVED** [review]

**Review by esauerbo — APPROVED** [review]

**Pull request merged**